### PR TITLE
[Calendar] Tune show meetings logic

### PR DESCRIPTION
**Pull request by KayMKM**
## Description

Show 3 meetings in one page.
Refactor meeting card and add content preview in card.
Show content when confirm the meeting creating

## Related Issue

https://fuselabs.visualstudio.com/Solutions/_queries/edit/26084/?triage=true

## Testing Steps

Try show meeting flow and read out one of meetings.
Try create a meeting with content.

## Checklist
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the appropriate unit tests
- [ ] I have updated related documentation

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
- [ ] A duplicate issue is filed to track future  work.

If you have updated responses or `.lu` files:
- [ ] All languages have been updated
- [ ] You have tested deployment with your new models

### CODE REVIEW

File: solutions/Virtual-Assistant/src/csharp/microsoft.bot.solutions/Util/CommonUtil.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ public class CommonUtil
 
         public const int MaxReadSize = 3;
 
-        public const int MaxDisplaySize = 6;
+        public const int MaxDisplaySize = 3;
```
Comment (lauren-mills) [inline solutions/Virtual-Assistant/src/csharp/microsoft.bot.solutions/Util/CommonUtil.cs]: It seems like these values don't belong in the microsoft.bot.solutions project. Can they go in the configuration in the skill appsettings? I think the todoskill does this today.

**Review by lauren-mills — CHANGES_REQUESTED** [review]

File: solutions/Virtual-Assistant/src/csharp/skills/todoskill/Dialogs/Main/MainDialog.cs [hunk 2]
```diff
@@ -318,6 +318,10 @@ private void RegisterDialogs()
         private void InitializeConfig(ToDoSkillState state)
         {
             // Initialize PageSize, ReadSize and TaskServiceType when the first input comes.
+            // workaround for UT failed
+            const int MaxDisplaySize = 6;
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/todoskill/Dialogs/Main/MainDialog.cs]: same as Lauren's comment. these should be from config as well.

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: solutions/Virtual-Assistant/src/csharp/microsoft.bot.solutions/Util/CommonUtil.cs (hunk 1 above)
Comment (KayMKM) [reply solutions/Virtual-Assistant/src/csharp/microsoft.bot.solutions/Util/CommonUtil.cs]: Thanks for figuring out. I added this number in calendar skill. Will tell email and todo to move this number to their skills and then we will delete these numbers from microsoft.bot.solutions.

File: solutions/Virtual-Assistant/src/csharp/skills/todoskill/Dialogs/Main/MainDialog.cs (hunk 2 above)
Comment (KayMKM) [reply solutions/Virtual-Assistant/src/csharp/skills/todoskill/Dialogs/Main/MainDialog.cs]: Removed.

File: solutions/Virtual-Assistant/src/csharp/assistant/appsettings.json [hunk 3]
```diff
@@ -1,104 +1,105 @@
...
-        "IPA.Timezone"
-      ],
-      "configuration": {
-        "googleAppName": "",
-        "googleClientId": "",
-        "googleClientSecret": "",
-        "googleScopes": "https://www.googleapis.com/auth/calendar https://www.googleapis.com/auth/contacts"
-      }
+    "botFilePath": ".\\YOUR_BOT_PATH.bot",
+    "botFileSecret": "",
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/assistant/appsettings.json]: can't really tell what's changed in this file :(

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/appsettings.json [hunk 4]
```diff
@@ -1,31 +1,32 @@
...
-  },
-  "defaultLocale": "en-us",
-  "languageModels": {
-    "en": {
-      "botFilePath": ".\\LocaleConfigurations\\YOUR_LOCALE_PATH.bot",
-      "botFileSecret": ""
+    "botFilePath": ".\\CalendarSkill.bot",
+    "botFileSecret": "",
+    "ApplicationInsights": {
+        "InstrumentationKey": ""
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/calendarskill/appsettings.json]: same here

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Common/CalendarCommonUtil.cs [hunk 5]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CalendarSkill.Common
+{
+    public class CalendarCommonUtil
+    {
+        public const int MaxDisplaySize = 3;
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Common/CalendarCommonUtil.cs:10]: this should go to setting as well

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: solutions/Virtual-Assistant/src/csharp/assistant/appsettings.json (hunk 3 above)
Comment (KayMKM) [reply solutions/Virtual-Assistant/src/csharp/assistant/appsettings.json]: It should be have extra space here. My editor did automatic indentation. I will revert the space change. And the real change in this file is adding a argument of page size.

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Common/CalendarCommonUtil.cs (hunk 5 above)
Comment (KayMKM) [reply solutions/Virtual-Assistant/src/csharp/skills/calendarskill/Common/CalendarCommonUtil.cs:10]: this is a backup of setting. if we can't get the value from appsetting we will get it from here.

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/appsettings.json (hunk 4 above)
Comment (KayMKM) [reply solutions/Virtual-Assistant/src/csharp/skills/calendarskill/appsettings.json]: fixed

**Review by lzc850612 — APPROVED** [review]

**Pull request merged**